html/modules/custom/reliefweb_api/reliefweb_api.module

```diff
@@ -171,7 +171,7 @@ function reliefweb_api_execute(array $options, $entity_type_id) {
  * @param \Drupal\Core\Entity\EntityInterface $entity
  *   Entity that might require re-indexing.
  */
-function reliefweb_api_check_reindexing(EntityInterface &$entity) {
+function reliefweb_api_check_reindexing(EntityInterface $entity) {
```

Comment:
@lazysoundsystem FYI, the `&` is unnecessary because `$entity` already holds an identifier pointing at the same object as the one in hook_taxonomy_term_update so any `$entity->xxx` here is be done on the object.

See: https://www.php.net/manual/en/language.oop5.references.php